meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy

```diff
@@ -592,6 +657,8 @@ update() {
 
   print_green "Successfully updated Dasharo firmware"
 
+  exit 0
```

Comment:
@miczyg1 why `exit 0` here? Should we not reboot after successful update?

Reply:
Some leftover from debugging (didn't want my unit tests to perform a poweroff of my host machine)

Reply:
```suggestion
```